webapp/pages/index.spec.js

```diff
@@ -53,6 +54,7 @@ describe('PostIndex', () => {
         'auth/user': () => {
           return { id: 'u23' }
         },
+        'posts/currentPosts': () => [],
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-11-11T23:39:32Z" title="Tuesday, November 12th 2019, 12:39:32 am +01:00">Nov 12, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -53,6 +54,7 @@ describe('PostIndex', () => {
         'auth/user': () => {
           return { id: 'u23' }
         },
+        'posts/currentPosts': () => jest.fn(),
```


```suggestion
        'posts/currentPosts': () => [],
```

Reply:
> [<img alt="Mogge" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Mogge) **Authored by [Mogge](https://github.com/Mogge)**
_<time datetime="2019-11-12T10:21:40Z" title="Tuesday, November 12th 2019, 11:21:40 am +01:00">Nov 12, 2019</time>_

---


done